StyleCop.Analyzers/StyleCop.Analyzers.Test/LayoutRules/SA1518UnitTests.cs

```diff
@@ -0,0 +1,152 @@
+﻿using Microsoft.CodeAnalysis;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using StyleCop.Analyzers.LayoutRules;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using TestHelper;
+using Microsoft.CodeAnalysis.Diagnostics;
+using Microsoft.CodeAnalysis.CodeFixes;
+
+namespace StyleCop.Analyzers.Test.LayoutRules
+{
+    /// <summary>
+    /// Unit tests for the SA1518: CodeMustNotContainBlankLinesAtEndOfFile
+    /// </summary>
+    [TestClass]
+    public class SA1518UnitTests : CodeFixVerifier
+    {
+        private const string DiagnosticId = SA1518CodeMustNotContainBlankLinesAtEndOfFile.DiagnosticId;
+
+        private const string BaseCode = @"using System.Diagnostics;
+public class Foo
+{
+    public void Bar(int i)
+    {
+        Debug.Assert(true);
+    }
+}";
+        
+        /// <summary>
+        /// Verifies that the analyzer will properly handle an empty source.
+        /// </summary>
+        /// <returns></returns>
+        [TestMethod, TestCategory("MaintainabilityRules/SA1518")]
```

Comment:
We have not used TestCategory anywhere else. Maybe we should either revert this back or add this at the other tests as well. Does this make these unit tasts stand out e.g. in the test explorer?